### ISSUE TITLE
The remote server returned an error: (302) Found. (Krita)

### ISSUE THREAD

**Issue by GiorgiBeriashvili** [lukesampson/scoop-extras#2932]
## Description

If I try to install Krita (both 32-bit and 64-bit versions) through scoop, I get this result:

Install command: `scoop install krita -a 64bit`

```sh
Installing 'krita' (4.2.7.1) [64bit]
The remote server returned an error: (302) Found.
URL http://download.kde.org/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z is not valid
```
I searched about the error on the internet and I came across this issue: https://github.com/lukesampson/scoop/issues/2129

@r15ch13 commented:

> Yeah, this happens because they redirect to mirrors without HTTPS. Retrying it is the only solution. (I hope letsencrypt gets more reach...)

This led me to visit the [mirror list](https://download.kde.org/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe.mirrorlist) page of Krita.

All of the redirects happen on **http** based websites. Some of them are actually **https**, but whether I put http or https variant of download.kde.org, - the URL redirection takes place and download fails due to error 302.

### Manual Workaround

I selected a preferred mirror URL instead of the base URL and replaced it inside [bucket/krita.json](https://github.com/lukesampson/scoop-extras/blob/master/bucket/krita.json):

```json
"architecture": {
    "64bit": {
        "url": "https://mirrors.mit.edu/kde/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z",
        "hash": "31538a959e7b271cf7c166f4e1e6162e5958e9b54055c4ecb1f50335fab2f01f"
    },
```

After this edit, `scoop install krita -a 64bit` worked and scoop successfully installed Krita on the system.

### Solution

Can there be an automated workaround to this issue?

I am thinking on making a pull request to put a fixed URL from the [Official KDE Mirrors](https://download.kde.org/extra/download-mirrors.html) list.

For example: https://mirrors.mit.edu/kde/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z

Thank you!

**Comment by AyoungDukie** [comment]
This does still appear to be an issue with 4.3.0; following the recommended workaround resolved the issue for me (ie going to the <download link>.mirrorlist, finding an appropriate mirror, then replacing the url in krita.json

**Comment by Ash258** [comment]
Fixed with lukesampson/scoop@e6b355eae093df70db85a6fc7850f6568bb6e7d9
